code/modules/mob/living/carbon/alien/organs.dm

```diff
@@ -197,23 +197,14 @@
 	for(var/atom/movable/thing as anything in stomach_contents)
 		if(!digestable_cache[thing.type])
 			continue
-		thing.reagents.trans_to(src, 4)
-
-		if(isliving(thing))
-			var/mob/living/lad = thing
-			lad.adjustBruteLoss(6)
-		else if(!thing.reagents.total_volume) // Mobs can't get dusted like this, too important
-			qdel(thing)
+		thing.acid_act(75, 10)
 
 /obj/item/organ/internal/stomach/alien/proc/consume_thing(atom/movable/thing)
 	RegisterSignal(thing, COMSIG_MOVABLE_MOVED, PROC_REF(content_moved))
 	RegisterSignal(thing, COMSIG_QDELETING, PROC_REF(content_deleted))
 	if(isliving(thing))
 		var/mob/living/lad = thing
 		RegisterSignal(thing, COMSIG_LIVING_DEATH, PROC_REF(content_died))
-		if(lad.stat == DEAD)
-			qdel(lad)
-			return
```

Comment:
how does this uh, fix anything? this wasn't a bug

Reply:
this entire check is done in such a weird way that i'm positive it's deleting people, i got deleted by this code in a round as someone who got abducted by a xenomorph who was confused as to why I suddenly disappeared from them mid transit back to the base; removing the deletion code entirely resolves that, as acid_act() will handle destroying objects anyways in a method that's actually stoppable by the crew if they rescue the items

Reply:
Sorry for the late response, mentioned this over discord but you did not respond.

This isn't a bug. xenos are supposed to delete people in their stomach who are dead. This is in fact STILL the case, via the stat change signal, you just removed the safety that deletes mobs that are already dead on swallow. This is dumb, and it's unclear to me what you are trying to do or even what you think is going on here.